### PR TITLE
Fix flaky plots tab test

### PR DESCRIPTION
**Pull request by adamabeshouse**
]

### CODE REVIEW

File: end-to-end-tests/specs/screenshot.spec.js [hunk 1]
```diff
@@ -330,7 +330,7 @@ describe("plots tab screenshot tests", function() {
     it("plots tab molecular vs molecular same gene", function() {
         browser.execute(function() { resultsViewPlotsTab.onHorizontalAxisDataTypeSelect({ value: "MRNA_EXPRESSION" }); });
         browser.execute(function() { resultsViewPlotsTab.onHorizontalAxisDataSourceSelect({ value: "brca_tcga_mrna" }); });
-        browser.waitForExist('input[data-test="ViewCopyNumber"]');
+        browser.waitForExist('input[data-test="ViewCopyNumber"]', 10000);
```
[the remark this answers is not in the document]
Comment (alisman) [reply end-to-end-tests/specs/screenshot.spec.js]: this is the default (set in webdriver conf file) so I'd be surprised if this did anything

[the remark this answers is not in the document]
Comment (adamabeshouse) [reply end-to-end-tests/specs/screenshot.spec.js]: that makes sense to me. let me rerun the tests here on circleCi a couple of times and see what happens

[the remark this answers is not in the document]
Comment (alisman) [reply end-to-end-tests/specs/screenshot.spec.js]: any outcome here?

[the remark this answers is not in the document]
Comment (adamabeshouse) [reply end-to-end-tests/specs/screenshot.spec.js]: it failed D:

**Pull request closed without merging**